frontend/apps/reader/modules/readerlink.lua

```diff
@@ -790,7 +790,9 @@ function ReaderLink:onGotoLink(link, neglect_current_location, allow_footnote_po
             if not neglect_current_location then
                 self:addCurrentLocationToStack()
             end
+            local originating_page = self.ui.paging.current_page
             self.ui:handleEvent(Event:new("GotoPage", link.page + 1, link.pos))
+            Notification:notify(_("Link from page " .. originating_page), Notification.SOURCE_OTHER)
```

Comment:
Assuming my comment `(This is called by other modules (highlight, search) to jump to a xpointer,` does not apply when on a PDF, and that this is called only when really following a in-page link, fine with me.
(Dunno how hard it would be to use a marker in the margin like we do with credocuments, for consistency.)

Also, dunno if the notification text `Link from page 43` is clear enough ? We like these text short, but I would understand this better if it said something like : "Just followed link from page 43".

Reply:
Welcome back poire-z! Hope you had a nice holiday.

>Also, dunno if the notification text Link from page 43 is clear enough ?

My initial idea was also to write, "Followed link from page 43", which would also be fine. I eventually decided to shorten it because 98%+ of the time users should already be aware that they just tapped a link anyway, and the notification is only for confirmation that the link was tapped correctly, and for rare occurrences where a link was tapped unintentionally. So by the time a user accidentally taps a link and gets alerted by this, they should already be well familiar with this notification, where it came from and what it means. The text of the notification is not that important anway IMO; it could also just say "Followed link", but I thought we might as well pack some extra info in there (the originating page no.), in cases where it might be useful.